### DISCUSSION TITLE
如何查看节点之间互联是通过直连还是转发？

### DISCUSSION THREAD

**Discussion by thwblueboy** [opengnb/opengnb#28] [Q&A]
1. 如何查看节点之间互联是通过直连还是转发？我通过`gnb_ctl`只能看到节点是REACHABL，似乎看不出是直连还是通过fw节点转发
2. 如果我配置了多个fw节点，在默认配置下（`--multi-forward-type=simple-fault-tolerant`），会如何选择fw节点？

**Comment by gnbdev** [comment]
1、REACHABL 就是 point to point 所谓的直连
2、simple-fault-tolerant 就是容错模式，如果有多个forward节点，第一个如果故障了就会转到第二个。

**Reply by thwblueboy** [reply]
那请问通过fw中转的，能看出是使用哪一个fw节点中转的吗？

**Reply by gnbdev** [reply]
通过二次开发应该可以看到
